### ISSUE TITLE
auto-copy manifest assets not work when extension build

### ISSUE THREAD

**Issue by chansyawn** [crxjs/rollup-plugin-chrome-extension#361]
### Build tool

Vite

### Where do you see the problem?

- [ ] In the browser
- [X] In the terminal

### Describe the bug

Plugin cannot find the icons file path when executing `vite build`, but the file has been duplicated to the `dist` folder.
The plugin looks like trying to get the file from the root directory, should the code here be changed to `config.build.outDir`.
https://github.com/crxjs/rollup-plugin-chrome-extension/blob/41ee1542f7ee470f8acddda1df15d2569509e008/packages/vite-plugin/src/node/plugin-manifest.ts#L275

### Reproduction

file structure
```bash
├── public
│   ├── favicon.svg
│   └── icons
│       ├── icon128.png
│       ├── icon16.png
│       ├── icon32.png
│       └── icon48.png
└──src
```
manifest.config.ts
```typescript
export default defineManifest({
  //...
  action: {
    default_icon: {
      16: 'icons/icon16.png',
      32: 'icons/icon32.png',
      48: 'icons/icon48.png',
      128: 'icons/icon128.png',
    },
  },
  icons: {
    16: 'icons/icon16.png',
    32: 'icons/icon32.png',
    48: 'icons/icon48.png',
    128: 'icons/icon128.png',
  }
});
```

### Logs

```shell
> tsc && vite build

vite v2.9.8 building for production...
✓ 639 modules transformed.
[crx:manifest-post] ENOENT: no such file or directory, open '/home/****/project/icons/icon16.png'
error during build:
Error: ENOENT: no such file or directory, open '/home/****/project/icons/icon16.png'
```

### System Info

```shell
System:
    OS: Linux 5.10 Ubuntu 20.04 LTS (Focal Fossa)
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
    Memory: 4.70 GB / 7.66 GB
    Container: Yes
    Shell: 5.8 - /usr/bin/zsh
  Binaries:
    Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
    Yarn: 1.22.18 - ~/.nvm/versions/node/v16.15.0/bin/yarn
    npm: 8.8.0 - ~/.nvm/versions/node/v16.15.0/bin/npm
  npmPackages:
    vite: ^2.9.7 => 2.9.8
```

### Severity

annoyance

**Comment by yvywang** [comment]
I have the same problem, hope to repair it soon 🤔

**Comment by yvywang** [comment]
<img width="352" alt="image" src="https://user-images.githubusercontent.com/53296318/167403814-594d1cdc-f86f-4770-9202-b0b2b47398ce.png">
I found that the plugin `crx:manifest-post` had this logic, and maybe should discard `icons` ? 🤔   @jacksteamdev

**Comment by jacksteamdev** [comment]
I've added icons to an integration test to ensure that manifest assets are copied from either the project root or the public dir.

**Comment by jacksteamdev** [comment]
This bugfix was released to NPM in [@crxjs/vite-plugin@1.0.4](https://github.com/crxjs/rollup-plugin-chrome-extension/releases/tag/%40crxjs%2Fvite-plugin%401.0.4) :rocket:
